go-licenses.yaml

```diff
@@ -170,9 +170,9 @@ module:
       path: LICENSE
       # gopkg.in/yaml.v2 is port of a MIT licensed library
       spdxId: Apache-2.0 / MIT
-  - name: go.opencensus.io
-    version: v0.22.5
-    license:
-      path: LICENSE
-      url: https://github.com/census-instrumentation/opencensus-go/blob/v0.22.5/LICENSE
-      spdxId: Apache-2.0
+  # - name: go.opencensus.io
```

Comment:
@Bobgy can you share some knowledge about this file? Would would we need an override entry as such?